### PR TITLE
Change to PUT and better catch errors

### PR DESCRIPTION
**Pull request by ZobelOmry**
## Context

Some changes after integrating with the API gateway.
Better catch errors, remove token auth (not used ATM), change schema to be in par with the API gateway's

## Tests

- [x] log the logs

### CODE REVIEW

File: services/logger.ts [hunk 1]
```diff
@@ -28,11 +28,15 @@ class Logger {
...
-      }
-    });
+    })
+      .then((result) => {
+        if (!result.ok) {
+          console.warn("Logging to Strigo failed", { result });
+        }
+      })
+      .catch((error) => {
+        console.log("Logging to Strigo failed", { err: error });
```
Comment (ZobelOmry) [inline services/logger.ts:38]: warn

**Comment by nir0s** [comment]
I mean something like `log.info` instead of `logToRemote(level)` and to consolidate `logToRemote` and `logToConsole` to a single function with a flag.

**Comment by ZobelOmry** [comment]
You are reading the abstracted function. Take a closer look and you'll see it's as you want it to be @nir0s

File: services/logger.ts [hunk 2]
```diff
@@ -3,41 +3,40 @@ import * as configManager from "../src/modules/config/config";
...
```
Comment (nir0s) [inline services/logger.ts:33]: console.error?

**Review by nir0s — APPROVED** [review]
Small fix, and that's it.

**Pull request merged**